apps/src/maze/SpellingControls.jsx

```diff
@@ -0,0 +1,36 @@
+var msg = require('./locale');
+
+var SpellingControls = function (props) {
+  return (
+    <div id="spelling-table-wrapper">
+      <table id="spelling-table" className="float-right">
+        <tbody>
+          <tr>
+            <td className="spellingTextCell">{msg.word()}:</td>
+            <td className="spellingButtonCell">
+              <button id="searchWord" className="spellingButton" disabled>
+                <img src="/blockly/media/1x1.gif"/>
+                {props.searchWord}
+              </button>
+            </td>
+          </tr>
+          <tr>
+            <td className="spellingTextCell">{msg.youSpelled()}:</td>
+            <td className="spellingButtonCell">
+              <button id="currentWord" className="spellingButton" disabled>
+                <img src="/blockly/media/1x1.gif"/>
+                <span id="currentWordContents"/>
+              </button>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  );
+};
```

Comment:
For lots of these components, there are probably opportunities to move styling from css into inline JSX, but I'm not bothering with any of that for now.